gnomad/sample_qc/pipeline.py

```diff
@@ -179,7 +180,10 @@ def get_qc_mt(
     )
 
     if ld_r2 is not None:
-        qc_mt = qc_mt.persist()
+        if checkpoint_path:
+            qc_mt = qc_mt.checkpoint(checkpoint_path)
```

Comment:
Might want to add an `overwrite=True`. Also, let's add a logger that a checkpoint is happening.